api/accounts.md

```diff
@@ -4,4 +4,13 @@ The accounts API deals with authentication and username-related operations.
 
 ## `GET /accounts/checkuusername/<username>`
 
-Gets the status of a _username_, not to be confused with a _user_.  Returns a [username object](definitions/username_object.md).  
+Gets the status of a _username_, not to be confused with a _user_.  Returns:
+ 
+```json
+{
+  "username": String,
+  "msg": String
+}
+```
```

Comment:
Thanks so much! Just one nitpick: this definition should be in the same format used in existing parts of the docs, e.g. [comment_object.md](https://github.com/towerofnix/scratch-api-unofficial-docs/blob/master/api/definitions/comment_object.md). Something like this:

    ```
    {
        "username": /* The username requested */
        "msg": /* The availability status of this username - see below */
    }
    ```

Reply:
No problem - still learning the conventions of this documentation style.  Will fix.   😊